doc/source/ray-air/examples/pytorch_resnet_batch_prediction.ipynb

```diff
@@ -0,0 +1,311 @@
+    "- The BatchPredictor uses CPUs for inference by default, please specify `num_gpus_per_worker` if you want to use GPUs."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2023-02-11 14:38:27,442\tINFO batch_predictor.py:184 -- `num_gpus_per_worker` is set for `BatchPreditor`.Automatically enabling GPU prediction for this predictor. To disable set `use_gpu` to `False` in `BatchPredictor.predict`.\n",
+      "Map_Batches: 100%|██████████| 50/50 [00:05<00:00,  9.27it/s]\n",
+      "Map Progress (2 actors 1 pending): 100%|██████████| 9/9 [00:16<00:00,  1.82s/it]\n"
+     ]
+    }
+   ],
+   "source": [
+    "predictions = batch_predictor.predict(\n",
+    "    ds, feature_columns=[\"image\"], keep_columns=[\"label\"], batch_size=128, max_scoring_workers=3, num_gpus_per_worker=1\n",
+    ")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Result Evaluation and Saving\n",
+    "\n",
+    "`BatchPredictor.predict()` will return a ray dataset with a column of model output with key \"predictions\", and all columns specified in `keep_columns`.\n",
+    "\n",
+    "In this example, the output of the ResNet model is a 1000-dimensional tensor containing the logits of each class. We select the top-5 classes with the highest classification probabilities and calculate the Top-1 and Top-5 errors:\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Map_Batches: 100%|██████████| 9/9 [00:00<00:00, 139.29it/s]\n",
+      "Shuffle Map: 100%|██████████| 9/9 [00:00<00:00, 613.36it/s]\n",
+      "Shuffle Reduce: 100%|██████████| 1/1 [00:00<00:00, 88.26it/s]\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Top-1 error:  0.15100000000000002\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Shuffle Map: 100%|██████████| 9/9 [00:00<00:00, 699.70it/s]\n",
+      "Shuffle Reduce: 100%|██████████| 1/1 [00:00<00:00, 130.66it/s]\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Top-5 error:  0.010000000000000009\n"
+     ]
+    }
+   ],
+   "source": [
+    "def calculate_matches(batch):\n",
+    "    batch[\"top_5_pred\"] = batch[\"predictions\"].apply(lambda x: np.argsort(-x)[:5])\n",
+    "    batch[\"top_5_match\"] = batch.apply(lambda x: x[\"label\"] in x[\"top_5_pred\"], axis=1)\n",
+    "    batch[\"top_1_match\"] = batch.apply(lambda x: x[\"label\"] == x[\"top_5_pred\"][0], axis=1)\n",
+    "    return batch\n",
+    "\n",
+    "predictions = predictions.map_batches(calculate_matches)\n",
```

Comment:
Let's just make this explicit since we use numpy earlier.

```suggestion
    "predictions = predictions.map_batches(calculate_matches, batch_format="pandas")\n",
```